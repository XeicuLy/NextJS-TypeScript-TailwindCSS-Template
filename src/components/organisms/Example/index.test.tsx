import { render } from '@testing-library/react';
import { Example } from '.';

describe('organisms/Example', () => {
  it('renders correctly', () => {
    const { container } = render(<Example />);
    expect(container).toMatchSnapshot();
  });
});
