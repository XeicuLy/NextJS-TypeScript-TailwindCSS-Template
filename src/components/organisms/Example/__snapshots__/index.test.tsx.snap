// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`organisms/Example renders correctly 1`] = `
<div>
  <h1>
    Example
  </h1>
</div>
`;
